src/tests/stats/test_mean.f90

```diff
@@ -1,75 +1,90 @@
 program test_mean
-use stdlib_experimental_error, only: assert
-use stdlib_experimental_kinds, only: sp, dp, int32, int64
-use stdlib_experimental_io, only: loadtxt
-use stdlib_experimental_stats, only: mean
-implicit none
+  use stdlib_experimental_error, only: assert
+  use stdlib_experimental_kinds, only: sp, dp, int32, int64
+  use stdlib_experimental_io, only: loadtxt
+  use stdlib_experimental_stats, only: mean
+  implicit none
 
-real(sp), allocatable :: s(:, :)
-real(dp), allocatable :: d(:, :)
+  real(sp), parameter :: sptol = 1.2e-06_sp
+  real(dp), parameter :: dptol = 2.2e-15_dp
```

Comment:
Minor comment: why not use the function`epsilon()`?

Reply:
Yes, we could use that. But then some power of epsilon (e.g. ˙epsilon(1.0_dp)**(7.0_dp / 8.0_dp)`) as epsilon itself is too strict (if the difference of two numbers of the order of 1.0 is less than epsilon, than by the definition of epsilon it can not be represented anyway...)

Reply:
Thank you. Good as it is.